### PR TITLE
Enable multibox deployment script

### PR DESCRIPTION
**Pull request by Antonstockmarr**
The user can type --hub-deploy or --spoke-deploy to run deployment for one of the scale units.
The deployer class is made abstract, and is implemented by the SingleBoxDeployer and the ScaleUnitDeployer.

### CODE REVIEW

File: src/CLI/Program.cs [hunk 1]
```diff
@@ -26,32 +26,14 @@ public static async Task Main(string[] args)
             }
             catch (Exception ex)
             {
-                Console.WriteLine(ex);
+                Console.WriteLine(ex.Message);
```
Comment (AndreasHassing) [inline src/CLI/Program.cs:29]: Why only `Message`? We should keep printing the full exception, right? This does not give us a stack trace to work with in user-reported bugs.

**Comment by AndreasHassing** [comment]
How was this tested?

Added comments, looks good.

**Review by AndreasHassing — CHANGES_REQUESTED** [review]

**Comment by Antonstockmarr** [comment]
PR has been tested on the single one box by running the hub deploy and spoke deploy and validating that the environments are correctly set up. Have also been testing the different combinations of argument inputs.

File: src/CLI/Program.cs (hunk 1 above)
Comment (Antonstockmarr) [reply src/CLI/Program.cs:29]: I was thinking that the exception messages from the ArgumentParser were enough explanation, but that is probably a bold assumption. I will change it back.

**Review by AndreasHassing — APPROVED** [review]

**Pull request merged**